src/msgpack.cpp

```diff
@@ -25,6 +25,33 @@
 #include <migraphx/serialize.hpp>
 #include <msgpack.hpp>
 
+namespace migraphx {
+inline namespace MIGRAPHX_INLINE_NS {
+
+// Leave an extra byte for error checking
```

Comment:
Where and how this extra byte is being used?

Reply:
I assumed it would be clamped, I guess it will overflow so we cant check for overflow.

Reply:
Still it might be good to avoid fully saturated values just in case there is some issue.